src/panels/config/ha-panel-config.ts

```diff
@@ -75,6 +75,14 @@ export const configSections: { [name: string]: PageNavigation[] } = {
       core: true,
     },
   ],
+  lovelace: [
```

Comment:
I think that it should be a new section. That way we have a section per `<tabs-with-subpage>` ?

Reply:
It is a new section now?